templates/web/pages/org_form/org_form.tt.html

```diff
@@ -51,6 +68,24 @@ <h2>[% lang("organization_members") %]</h2>
 			[% END %]
 		</tbody>
 		</table>
+		<div>
+			<form id='hiddenInputForm' method="post" action="/cgi/org.pl">
```

Comment:
Why not put this around the table above where we display status, this is not a problem as those are the only values that can be modified ?
This method of duplicating the form is smart, but might be more difficult to debug.

Reply:
you didn't do it ? You would not need any JS if you do it.

Reply:
it seems you indeed have more than one form, but you can associate your input above to this form by using the "form" attribute on the input. see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#form
I would avoid javascript if possible.